<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visual SQL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/img/img/logo.png</normaloff>:/img/img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="Splitter_Main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>311</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QSplitter" name="Splitter_Top">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="Input_Filter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Filter databases and tables</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="Tree_Structure">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="tabKeyNavigation">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="iconSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="indentation">
            <number>12</number>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="uniformRowHeights">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QTabWidget" name="Tabs">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="Tab_Table">
            <attribute name="title">
             <string>Table</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="Tab_Data">
            <attribute name="title">
             <string>Data</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTableView" name="DataTable"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Tab_Query">
            <attribute name="title">
             <string>Custom query</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QTextEdit" name="Input_CustomQuery">
                <property name="contextMenuPolicy">
                 <enum>Qt::DefaultContextMenu</enum>
                </property>
               </widget>
               <widget class="QTableView" name="tableView"/>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QTextEdit" name="Input_Logs">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>80</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Main_MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu_File">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="Menu_File_Connections"/>
    <addaction name="Menu_File_ExecuteSQL_file"/>
    <addaction name="separator"/>
    <addaction name="Menu_File_Exit"/>
   </widget>
   <widget class="QMenu" name="Menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="Menu_Help_MySQL_documentation"/>
    <addaction name="Menu_Help_MariaDB_documentation"/>
    <addaction name="separator"/>
    <addaction name="Menu_Help_AboutVisual_SQL"/>
   </widget>
   <addaction name="Menu_File"/>
   <addaction name="Menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="Main_StatusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="Main_ToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Menu_File_Connections"/>
   <addaction name="Menu_File_ExecuteSQL_file"/>
   <addaction name="separator"/>
   <addaction name="Menu_Help_AboutVisual_SQL"/>
   <addaction name="Menu_File_Exit"/>
  </widget>
  <action name="Menu_File_ExecuteSQL_file">
   <property name="icon">
    <iconset theme="itmages-upload">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Execute SQL file</string>
   </property>
  </action>
  <action name="Menu_File_Exit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="Menu_Help_AboutVisual_SQL">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About Visual SQL</string>
   </property>
  </action>
  <action name="Menu_Help_MySQL_documentation">
   <property name="text">
    <string>&amp;MySQL documentation</string>
   </property>
  </action>
  <action name="Menu_Help_MariaDB_documentation">
   <property name="text">
    <string>MariaDB &amp;documentation</string>
   </property>
  </action>
  <action name="Menu_File_Connections">
   <property name="icon">
    <iconset theme="view-list-details">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Connections</string>
   </property>
  </action>
  <action name="actionDrop_database">
   <property name="text">
    <string>Drop database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Menu_File_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
